lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,9 +586,10 @@ module.exports = class MethodMappings {
 
         async sendKeysToElement(webElementOrId, value) {
           if (Array.isArray(value)) {
+            if (value.includes(undefined) || value.includes(null)){
+              throw new Error('sendKeys - each key must be a number or string');
```

Comment:
```suggestion
              throw TypeError('each key must be a number of string; got ' + value)
```

Reply:
fixed in #[8537819](https://github.com/nightwatchjs/nightwatch/pull/3115/commits/8537819be822cd53b66c0d237314ef2ce9c8a419)